app/controllers/results_submission_controller.rb

```diff
@@ -12,6 +12,28 @@ def new
     @results_validator.validate(@competition.id)
   end
 
+  def duplicate_checker
+    @competition = competition_from_params
+  end
+
+  def last_duplicate_checker_job
+    competition = competition_from_params
+
+    render status: :ok, json: competition.duplicate_checker_jobs.order(created_at: :desc).first
```

Comment:
You can implement this either as a `default_scope { order(created_at: :desc) }` on the `DuplicateCheckerJob` or as a scope on the `has_many :duplicate_checker_jobs, -> { order(...) }` on Competition

Reply:
Also, unrelated: I am not a big fan of "abusing" the technical `created_at` and `updated_at` timestamps for business logic. I know that we do that in other places of the code, but those are all legacy use cases that I would like to get rid of.

Why are you not using `start_time` and/or `end_time` here?